.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
@Soullivaneuh: Not sure about the exact Travis behaviour on `fast_finish`, and how it may change the tests beahviour : the [relative documentation](http://docs.travis-ci.com/user/build-configuration/#Fast-finishing) actually tells `a build will finish as soon as a job has failed`.

Have any personal experience on that, with related links to builds having a `fast_finish` marked configuration?